base/array.jl

```diff
@@ -175,6 +175,20 @@ false
 isbitsunion(u::Union) = (@_pure_meta; ccall(:jl_array_store_unboxed, Cint, (Any,), u) != Cint(0))
 isbitsunion(x) = false
 
+isptrelement(t::Type) = (@_pure_meta; ccall(:jl_array_store_unboxed, Cint, (Any,), t) == Cint(0))
```

Comment:
we already define this [here](https://github.com/JuliaLang/julia/blob/cef655a68a4e824cbe7cca582256d61aa72eb725/base/array.jl#L175), so maybe we should consolidate.

Reply:
It's not quite the same, since the other query is related to this query by the formula `isbitsunion(u) = u isa Union && isptrelement(u)`

Reply:
Right; I was mainly pointing out that maybe we should define a single `ccall` definition for this (and have `isbitsunion` call that), just to minimize the need to update things in two places if `jl_array_store_unboxed` ever changes.